### PR TITLE
revising data model for EDU Lesson procedures

### PR DESCRIPTION
**Pull request by stephiescastle**
### Checklist

- [x] Include a description of your pull request and instructions for the reviewer to verify your work.
- [x] Link to the issue if this PR is issue-specific.
- [x] Create/update the corresponding story if this includes a UI component.
- [ ] Create/update documentation. If not included, tell us why.
- [x] List the environments / browsers in which you tested your changes.
- [x] Tests, linting, or other required checks are passing.
- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../releases); good ones make that easier to scan.
  - PRs will be broadly categorized in the change log, but for even easier scanning, consider prefixing with a component name or other useful categorization, e.g., "BaseButton: fix layout bug", or "Storybook: Update dependencies".
- [x] PR has been tagged with a [SemVer](https://semver.org/) label and a [general category label](https://github.com/nasa-jpl/explorer-1/blob/52994b671411f55961d7beb8851d4580ac3f434f/.github/release-drafter.config.yml#L21-L39), or skip-changelog.
  - These tags are used to do the aforementioned broad categorization in the change log and determine what the next release's version number should be.
  - [Release Drafter](https://github.com/marketplace/actions/release-drafter) will attempt to do the category labeling for you! Please double-check its work.

## Description

Supports https://github.com/nasa-jpl/www/issues/413

- Revises data structure for lesson procedures

### Caveats

- When using steps numbering, the first item in the "step" should be a rich text block (or similarly narrow block), otherwise the number will be covered up by wider media blocks.

## Instructions to test

1. `make vue-storybook`
2. View the PageEduLessonDetail story

### Operating System

- [x] macOS
- [ ] iOS
- [ ] iPadOS
- [ ] Windows

### Browser

- [x] Chrome
- [x] Firefox ESR
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: packages/vue/src/templates/edu/PageEduLesson/PageEduLesson.stories.js [hunk 1]
```diff
@@ -169,7 +169,49 @@ export const BaseStory = {
...
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            }
+          ]
         },
```
Comment (Scotchester) [inline packages/vue/src/templates/edu/PageEduLesson/PageEduLesson.stories.js:215]: I think the two `procedures` items below here should probably be removed? They don't appear to be causing a problem, but they don't do anything, either.

File: packages/vue/src/templates/edu/PageEduLesson/PageEduLesson.stories.js [hunk 2]
```diff
@@ -169,7 +169,49 @@ export const BaseStory = {
...
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            }
+          ]
+        },
+        {
+          sectionHeading: 'Section Heading 3',
+          stepsNumbering: true,
```
Comment (Scotchester) [inline packages/vue/src/templates/edu/PageEduLesson/PageEduLesson.stories.js:203]: Let's test this one with no numbering.

```suggestion
          stepsNumbering: false,
```

**Comment by Scotchester** [comment]
> When using steps numbering, the first item in the "step" should be a rich text block (or similarly narrow block), otherwise the number will be covered up by wider media blocks.

@kimorr @otalutz Do you think this will be an acceptable limitation? Will most steps start with text?

Headings at levels larger than 4 look pretty funky due to the baselines of the number and the heading being far apart, but most likely and heading within one of these steps should be an H4, since "Procedures" is an H2 and procedure section headings would be H3.

**Review by Scotchester — CHANGES_REQUESTED** [review]
This is working pretty well for me! Just a couple needed changes in the stories, I think.

**Comment by stephiescastle** [comment]
> This is working pretty well for me! Just a couple needed changes in the stories, I think.

Made the changes! Forgot to do a lot of cleanup before opening the PR 🤦 thanks for the review!

File: packages/vue/src/templates/edu/PageEduLesson/PageEduLessonSection.vue [hunk 3]
```diff
@@ -68,23 +70,51 @@ const anchorId = computed(() => {
     />
     <template v-else-if="procedures?.length">
       <template
-        v-for="(item, index) in procedures"
-        :key="index"
+        v-for="(section, _index) in procedures"
+        :key="_index"
```
Comment (Scotchester) [inline packages/vue/src/templates/edu/PageEduLesson/PageEduLessonSection.vue:74]: It appears that it doesn't like this `_index` not being used now?

Comment (Scotchester) [reply packages/vue/src/templates/edu/PageEduLesson/PageEduLessonSection.vue:74]: Ah, you beat me :)

**Review by Scotchester — APPROVED** [review]
LGTM! We can revisit if my question for Kim and Ota necessitates it.

**Pull request merged**